style.css

```diff
@@ -462,3 +504,438 @@ body {
   margin-bottom: 100px;
   width: 130px;
 }
+
+@media only screen and (min-width: 768px) {
```

Comment:
It looks great 🎉 very well done, but for some reason, your design is not taking the whole width of the screen. Take a look 👇🏽 
![Screenshot 2023-02-24 at 08 18 38](https://user-images.githubusercontent.com/45739963/221188326-b13afeee-a7c7-462c-ad72-55b1c92055af.png)

---------

![Screenshot 2023-02-24 at 08 19 55](https://user-images.githubusercontent.com/45739963/221188606-dd2cec40-fc62-46ed-a085-2d1663878b9f.png)